addon/components/frost-table-cell.js

```diff
@@ -34,5 +36,10 @@ export default Component.extend({
   // == Actions ===============================================================
 
   actions: {
+    handleCallback (...args) {
```

Comment:
I think all you would have to do in `frost-table` is define an `onCallback` that gets defaulted to `function () {}` in `getDefaultProps()`